src/components/ScholarshipForm.jsx

```diff
@@ -28,17 +29,7 @@ function ScholarshipForm({ setSubmissionAlert }) {
   const user = firebase.auth().currentUser;
 
   const formik = useFormik({
-    initialValues: {
-      name: '',
-      deadline: null,
-      description: '',
-      amount: {
-        type: null,
-        min: 0,
-        max: 0,
-      },
-      website: '',
-    },
+    initialValues: InitialValues,
```

Comment:
Why is this being refactored? The initial values are not specific to Formik, they're specific to the ScholarshipForm. This should either be derivable in this component or passed in as a prop.